code/__DEFINES/atmospherics.dm

```diff
@@ -537,3 +537,8 @@ GLOBAL_LIST_INIT(pipe_paint_colors, sortList(list(
 
 #define MIASMA_CORPSE_MOLES 0.02
 #define MIASMA_GIBS_MOLES 0.005
+
+//flags for N2O and Healium euphoria moodlets
```

Comment:
Nitpick, but they're not flags, calling them defines will work fine here

Reply:
changed